Formula/x11vnc.rb

```diff
@@ -50,7 +50,7 @@ def install
 --- a/x11vnc/solid.c
 +++ b/x11vnc/solid.c
 @@ -177,7 +177,7 @@ unsigned long get_pixel(char *color) {
- 
+
```

Comment:
ooOOOOooh 😮 That's awesome! I went through each patch and said 'n' to the white space edit. Now I'll use -p all the time 👍🏾 Thank you!